doc/application/index.rst

```diff
@@ -723,7 +723,17 @@ its location through the ``DTS_ROOT`` CMake Cache variable:
    :goals: build
    :compact:
 
-You can also define the variable in the application
+The default sequence of ``DTS_ROOT`` values is
+``APPLICATION_SOURCE_DIR``, ``BOARD_DIR``, and ``ZEPHYR_BASE``.  For
+those `CMake`_ will add the following paths to be searched when
+processing Device Tree overlays::
+
+    include
+    dts/common
+    dts/${ARCH}
+    dts
```

Comment:
We should not have to repeat what happens when a directory is in the DTS_ROOT as this is already stated above.

> the other directories contain files that can be included from DT sources.

If you feel this is not being conveyed appropriately then we should update the original documentation instead of re-iterating it.

Reply:
I've updated the documentation to use your proposed text, and changed the order and content to match what I think `dts.cmake` is doing.  Please verify.

I also updated the spelling of "Device Tree" in the nearby lines to match the most common spelling in that file.